style.css

```diff
@@ -0,0 +1,522 @@
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1em;
+  color: black;
+  text-align: center;
+  position: relative;
+  margin: 20px 30px;
+}
+
+/*** ACCORDION ****/
+
+.accordion-question{
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.8em;
+  color: black;
+  text-align: center;
+}
+
+.accordion{
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  position: relative;
+}
+
+.accordion-h3{
+  font-family: 'Montserrat', sans-serif;
+  margin-left: 5px;
+  font-size: 1em;
+  text-align: left;
+}
+
+.question{
+  width: 70%;
+  display: flex;
+  justify-content: space-between;
+  height: auto;
+  text-align: justify;
+  border: solid 2px black;
+  cursor: pointer;
+  border: none;
+  outline: none;
+  border-bottom: solid 0.5px #555;
+  transition: 0.4s;
+}
+
+.active, .question:hover{
+  background-color: #ecded5
+}
+
+
+.active, .answer{
+  background-color: #F2F2F2;
+}
+
+.answer{
+  width: 70%;
+  height: 0;
+  text-align: justify;
+  overflow:hidden;
+  transition: height 0.3s ease;
+  margin: 5px 0;
+}
+
+.answer-p{
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1em;
+  text-align: justify;
+}
+
+.answer-list{
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1em;
+  text-align: justify;
+}
+
+.accordion-image{
+  width:100%;
+  justify-items: center;
+}
+
+span.italic{
+  font-style: italic;
+}
+
+.active{
+  height: 270px; 
+}
+
+.active-image{
+  height: 230px;
+}
+
+
+/*** SIGN UP FORM ****/
+
+
+form{
+  box-sizing: border-box;
+  width:50%;
+}
+
+.field-container{
+  margin-top:5px;
+  width:70%;
+}
+
+.the-form{
+  box-sizing: border-box;
+  background-color: #ecded5;
+  display: flex;
+  justify-content: center;
+  margin: 60px 0px 0px 0px;
+}
+
+.p-form-main{
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1em;
+  color: black;
+  text-align: justify;
+}
+
+.p-form{
+  box-sizing: border-box;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.1em;
+  color: black;
+  font-weight: 900;
+  text-align: left;
+  margin: 15px 5px;
+}
+
+.p-form-small{
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1em;
+  font-style: bold;
+}
+
+.amount-container{
+  display:inline-flex;
+}
+
+.amount{
+  box-sizing: border-box;
+  margin: 5px;
+}
+
+/* radio buttons styling */
+
+.radio-button-container{
+  display: block;
+  position: relative;
+  padding-left: 25px;
+  margin: 10px 5px;
+  font-size: 16px;
+  user-select: none;
+}
+
+
+.radio-button-container input{
+  position: absolute;
+  opacity: 0;
+  cursor: pointer;
+  height: 16px;
+  width: 16px;
+  transform: translate(-180%, -255%);
+  z-index: 1;
+}
+
+.custom-radio{
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 17px;
+  width: 17px;
+  background-color: #eee;
+  cursor: pointer;
+  border-radius: 50%;
+}
+
+.radio-button-container:hover input ~ .custom-radio{
+  background-color: #444444;
+}
+
+.radio-button-container input:checked ~ .custom-radio{
+  background-color: #444444
+} 
```

Comment:
Consider having the hover effect lighter than the checked one to clarify the difference